agent_api_rest/postman/ssi-agent.postman_collection.json

```diff
@@ -546,6 +546,53 @@
 						}
 					},
 					"response": []
+				},
+				{
+					"name": "notification",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"pm.test(\"Status code is 204 No Content\", function () {",
+									"    pm.response.to.have.status(204);",
+									"});"
+								],
+								"type": "text/javascript",
+								"packages": {}
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "Authorization",
+								"value": "Bearer {{ACCESS_TOKEN}}",
+								"type": "text"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n  \"notification_id\": \"{{NOTIFICATION_ID}}\",\n  \"event\": \"credential_accepted\",\n  \"event_description\": \"User accepted the credential in wallet\"\n}",
```

Comment:
`NOTIFICATION_ID` is used here, but it is never set in during the flow. I think it would be nice if you'd add a "Tests" script to the `{{HOST}}/openid4vci/credential` Credential request that catches the `notification_id` from the Credential response in a similar fashion as how we catch the `ACCESS_TOKEN` value from the Token response:
![image](https://github.com/user-attachments/assets/13666b72-aedb-4ab1-a355-464174fe6622)